src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,7 +2800,7 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-macos`
Function Pipeline Behaviour.Output from Named Blocks.does not permit output from dispose{}
```powershell
Expected $null or empty, but got 11.
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Language/Scripting/PipelineBehaviour.Tests.ps1: line 21
21:             & { dispose { 11 } } | Should -BeNullOrEmpty
```